iosApp/iosApp/iosApp.entitlements

```diff
@@ -8,5 +8,9 @@
 	<true/>
 	<key>com.apple.developer.usernotifications.time-sensitive</key>
 	<true/>
+	<key>com.apple.security.application-groups</key>
+	<array>
+		<string>group.ac.at.lbg.dhp.more.group</string>
+	</array>
```

Comment:
What is this used for? Should this be a proper package name?

Reply:
Needed to group both the iOS app and the notification service extension and give them a shared UserDefaults KeyValue Store, to access the current number of unread notifications